puppet/zulip_ops/templates/iptables/rules.v4.erb

```diff
@@ -39,6 +39,9 @@
 -A INPUT -p tcp --dport  443 -j ACCEPT
 -A INPUT -p tcp --dport 5432 -j ACCEPT
 
+# node_exporter
```

Comment:
Let's include "prometheus" in this comment to make it clearer for someone not familiar with what `node_exporter` is.